### PR TITLE
Changes to some command line options

### PR DESCRIPTION
**Pull request by khou2020**
-x for I/O strategy
pnc -> pnetcdf
adios2 -> adios
hdf5 -> hdf5_rearranger
hdf5_logvol -> hdf5_log
hdf5_multi -> hdf5_mv

Internal class name unchanged.

### CODE REVIEW

File: src/e3sm_io.c [hunk 1]
```diff
@@ -156,17 +156,17 @@ int main (int argc, char **argv) {
...
-                    cfg.api = hdf5_multi;
+                    cfg.api = hdf5_mv;
 #else
                     RET_ERR ("The HDF5 used does not support multi-dataset write")
 #endif
-                } else if (strcmp (optarg, "hdf5_logvol") == 0) {
+                } else if (strcmp (optarg, "hdf5_mv") == 0) {
 #ifdef ENABLE_LOGVOL
-                    cfg.api = hdf5_logvol;
+                    cfg.api = hdf5_mv;
```
Comment (wkliao) [inline src/e3sm_io.c:169]: Is this a mistake?
i.e. within ifdef ENABLE_LOGVOL

**Comment by wkliao** [comment]
rename 'hdf5_rearranger' to 'hdf5_ra'.

The README.md file needs to be updated with these option changes.

**Comment by khou2020** [comment]
fixed

**Comment by wkliao** [comment]
* What is the difference between adios and adios_bp3?
* README.md must be updated for all new or updated command-line options with
   descriptions of what they mean.

**Comment by khou2020** [comment]
fixed

**Comment by wkliao** [comment]
I do not see the descriptions in README.md

**Comment by wkliao** [comment]
Why still keep adios_bp3?

**Pull request merged**